tests/test_helpers.py

```diff
@@ -32,7 +32,7 @@
     (["helper", "l_projecta", "--verbose", "-p", "ascopatz"],
      "20ly_newprojectum\n20sb_firstprojectum\n"
      ),
-    (["helper", "l_grants", "-d", "2020-05-25"],
+    (["helper", "l_grants", "-c", "-d", "2020-05-25"],
```

Comment:
nice!  Actually it is this test that we needed the date for mostly, but I think I may also ne happy that we added UC3!